src/main/scala/za/co/absa/hyperdrive/trigger/scheduler/utilities/Configs.scala

```diff
@@ -100,4 +100,9 @@ object SparkExecutorConfig {
     Try(Configs.conf.getString("sparkYarnSink.filesToDeploy").split(",").toSeq).getOrElse(Seq.empty[String])
   def getAdditionalConfs: Map[String, String] =
     Configs.getMapFromConf("sparkYarnSink.additionalConfs")
+}
+
+object JobDefinitionConfig {
+  val KeysToMerge = Set("spark.executor.extraJavaOptions", "spark.driver.extraJavaOptions")
```

Comment:
I would move this to SparkExecutorConfig

Reply:
Yes, but it's not only about the SparkExecutor. It's also about the JobTemplate and JobDefinition. But anyway I can move it

Reply:
Yes you are right. Leave it here